build/common.sh

```diff
@@ -311,6 +311,7 @@ function kube::build::build_image() {
     api
     build
     cmd
+    contrib/mesos
```

Comment:
IIRC, NOT having this here causes building a release to fail... 
I don't remember why off the top of my head, tho.